docs/Setting_3rd_party_cert_environment.md

```diff
@@ -0,0 +1,61 @@
+# Set 3rd party cert environment
+
+##Disclaimer
+This doc intention is to help developer with building environment using 3rd party certificate,
+Used for **TESTING PURPOSE** and by no means provide an instruction on how to set up a production environment
+
+## Motivation
+
+Every certificate has to be signed by some CA (Certification Authority). There are many
+CA's out there. Some are big, like Google or Amazon, and some are small CA's.
+One can also create it own private CA and signed it own certificates.
+Many Companies (mostly big companies) choose to create their own CA and issue 
+their own certificates. This doc purpose is to be a walk-through on how to issue 
+your own certificate and to configure your env to work with your CA and issued certificate.
+
+## General information
+- Python comes with a list of trusted CAs. Every certificate signed by those CAs 
+will be considered as trusted certificate. Those CAs are located in a file called cacerts.pem. 
+  To get the path of this file, run in python `import certifi; certifi.where()` 
+
+## Walk-through
+### Creating your own CA
+1) Create your own folder to store the CA and open a terminal
+2) Create a rootCA encryption key: `openssl genrsa -des3 -out CA-NAME.key 2048`
+3) Create a rootCA cert:  `openssl req -x509 -new -nodes -key CA-NAME.key -sha256 -days 1825 -out CA-NAME.pem`
+    * Note you will be asked to prompt some information about the CA this values are not affecting 
+      the process by any way
+
+### Sign certificate for conjur server
+1) Create cert key: `openssl genrsa -out CERT-NAME.key 2048`
+2) Generate csr (certificate signing request) : `openssl req -new -key CERT-NAME.key -out CERT-NAME.csr`
+    * Note that you will need to provide information. the only thing that is important is the Common_name section.
+    The value for this section should be your conjur url.
```

Comment:
```suggestion
    The value for this section should be your Conjur URL.
```

what happens if there is a LB? Should it still be the URL for the conjur master? Should it be the DNS?

Reply:
should be the lb. meaning this shoould be the dns that we use to address conjur